.github/workflows/deploy.yml

```diff
@@ -0,0 +1,18 @@
+name: Deploy
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - run: npm install
```

Comment:
You should add a test step here so it does not deploy if tests fail.

Reply:
Yes! I'll wait with updating here until we have a solution for the credentials.